lib/find-python.js

```diff
@@ -188,15 +188,10 @@ PythonFinder.prototype = {
   // Distributions of Python on Windows by default install with the "py.exe"
   // Python launcher which is more likely to exist than the Python executable
   // being in the $PATH.
-  // Because the Python launcher supports all versions of Python, we have to
-  // explicitly request a Python 2 version. This is done by supplying "-2" as
-  // the first command line argument. Since "py.exe -2" would be an invalid
-  // executable for "execFile", we have to use the launcher to figure out
-  // where the actual "python.exe" executable is located.
   checkPyLauncher: function checkPyLauncher (errorCallback) {
     this.log.verbose(
-      `- executing "${this.pyLauncher}" to get Python 2 executable path`)
-    this.run(this.pyLauncher, ['-2', ...this.argsExecutable], false,
+      `- executing "${this.pyLauncher}" to get Python executable path`)
+    this.run(this.pyLauncher, [...this.argsExecutable], false,
```

Comment:
I don't think `[...this.argsExecutable]` needs to be an array anymore. I'm not sure if the ... is needed, either.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

https://stackoverflow.com/questions/31048953/what-do-these-three-dots-in-react-do

Reply:
Small moves please.  Leave this PR as is and allow maintainers time to review it as is.

Reply:
Tests pass with the following: 
```suggestion
    this.run(this.pyLauncher, this.argsExecutable, false,
```

Reply:
Will wait for updates from reviewers, thanks.

Reply:
yeah, go with your suggestion, it's correct 👍